tests/nested/manual/hybrid-remodel/task.yaml

```diff
@@ -4,7 +4,8 @@ details: |
   This test remodels on a hybrid system to install a new kernel snap and new
   application snaps.
 
-systems: [ubuntu-22.04-64, ubuntu-24.04-64]
+#TODO: enable for ubuntu 24.04
+systems: [ubuntu-22.04-64]
 
```

Comment:
I actually would prefer to let it fail and research, otherwise this will fall out of our radar.